manifest.json.template

```diff
@@ -64,13 +64,13 @@
       "explode": "true"
     },
     "org.zowe.explorer-jes": {
-      "version": "~1.0.0-STAGING"
+      "version": "~1.0.1-STAGING"
     },
     "org.zowe.explorer-mvs": {
-      "version": "~1.0.0-STAGING"
+      "version": "~1.0.1-STAGING"
     },
     "org.zowe.explorer-uss": {
-      "version": "~1.0.0-STAGING"
+      "version": "~1.0.1-STAGING"
```

Comment:
These changes aren't actually necessary AFAIK. I believe the way Jack set it up is that ~1.0.0 takes the latest in the 1.0.x